src/components/ReportBanner.tsx

```diff
@@ -86,7 +86,7 @@ export const ReportBanner = (props: ReportBannerProps) => {
   React.useEffect(() => {
     let failed = 0;
     filesProcessed?.forEach((file) => {
-      if (file.state === 'Failed' || file.state === 'Missing' || !file.bimFileExists || !file.fileExists) {
```

Comment:
what does this help with? And can there be a scenario where those booleans are not set but `state` is present?

Reply:
This started out fixing one issue I found testing this locally inside the portal. The logic for this should be:
```
if (file.state === 'Failed' || file.state === 'Missing' || (!file.bimFileExists && !file.fileExists))
```
Then I decided to removed the `state` checks since a) we are not checking for them inside the table, so we have a chance to misrepresent the amount of failures we have in the banner. and b) I have never seen the `state` property set. 

Maybe we should add the `state` checks to the Table instead of removing them here, just in case?

Reply:
I think we might want to ask around and find out what's the deal with `state`.

When [`FilesTable`](https://github.com/iTwin/synchronization-report-react/blob/main/src/components/FilesTable.tsx#L133) was built, we left this comment because we were unsure what to do.
![image](https://user-images.githubusercontent.com/9084735/156779488-c3bcc9ec-c4de-40cb-939b-3ac67dbfa794.png)

Reply:
After some conversations with the BADGERs team, it looks like the `state` property is currently unused for this kind of application.